openmetadata-ui/src/main/resources/ui/src/utils/TourUtils.tsx

```diff
@@ -210,7 +193,6 @@ export const getSteps = (value: string, clearSearchTerm: () => void) => {
           Click on the <strong>&quot;Manage&quot;</strong> tab
         </p>
       ),
-      position: [440, 240],
```

Comment:
Is the positioning for each of theses handled from within the library?

Reply:
Yes

Reply:
Instead of giving absolute position, we are not just passing priority order to the component like `['top', 'bottom', 'left', 'right']` and library will take care of it based on available rect and offsets